libraroo-backend/README.md

```diff
@@ -85,6 +85,8 @@ Place in project root
 ```
 DATABASE_CONNECTION_STRING=
 JWT_SECRET=
+GOOGLE_API_KEY=
```

Comment:
Change to GOOGLE_API_KEY=https://www.googleapis.com as this isnt really a secret

Reply:
yaanu add the value to the readme